src/hevm/src/EVM/SMT.hs

```diff
@@ -254,6 +254,41 @@ prelude =  (flip SMT2) mempty $ fmap (T.drop 2) . T.lines $ [i|
   (declare-const chainid Word)
   (declare-const basefee Word)
 
+  ; macros
+  (define-fun signext ( (sz (_ BitVec 256)) (val (_ BitVec 256))) (_ BitVec 256) (
```

Comment:
We actually should be extending by `sz + 1` (weirdly)....

Reply:
Sorry. Fixed :) Chris said the reason is: "we extend byte X" I.e. it's not "size", but "the byte to extend". I now renamed it to "b" rather than "size", since it's more appropriate.